### PR TITLE
Products and Categories Pages Added

### PR DESCRIPTION
**Pull request by WishaEman**

### CODE REVIEW

File: my-app/package.json [hunk 1]
```diff
@@ -8,8 +8,11 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^1.4.0",
     "bootstrap": "^5.3.1",
+    "jquery": "^3.7.0",
```
Comment (mubbsharanwar) [inline my-app/package.json:12]: why do you need to install jquery?

File: my-app/public/index.html [hunk 2]
```diff
@@ -5,6 +5,11 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
+    <link
```
Comment (mubbsharanwar) [inline my-app/public/index.html:8]: Can we install font-awesome dependency instead getting from cdn?

File: my-app/src/App.js
Comment (mubbsharanwar) [inline my-app/src/App.js:1]: We can delete this file if not needed

File: my-app/src/Shared/Footer/Footer.jsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+export default function Footer() {
+  return(
+      <footer className="bg-light text-center text-lg-start">
+        <div className="text-center text-dark p-3" style={{backgroundColor: 'rgba(0, 0, 0, 0.2)'}}>
```
Comment (mubbsharanwar) [inline my-app/src/Shared/Footer/Footer.jsx]: Try to do style in css file and use class here.

File: my-app/src/Shared/Subcategory/Subcategory.jsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    if (onSubcategoryClick) {
+      onSubcategoryClick(subcategory.title);
+    }
+  };
+
+  return (
+    <div>
+          <NavDropdown title={category?.title} id="categoryDropdown">
+            {subcategories && subcategories.map(subcategory => (
+              <NavDropdown.Item key={subcategory.id} href="#" onClick={() => handleSubcategoryClick(subcategory)}>
```
Comment (mubbsharanwar) [inline my-app/src/Shared/Subcategory/Subcategory.jsx]:
```suggestion
              <NavDropdown.Item key={subcategory.id} onClick={() => handleSubcategoryClick(subcategory)}>
```

Comment (mubbsharanwar) [inline my-app/src/Shared/Subcategory/Subcategory.jsx]: we can query products against subcategory id as we are making it a foreign key in products, right?  we can only pass id in `handleSubcategoryClick` instead object.

File: my-app/src/modules/Home/Home.css
Comment (mubbsharanwar) [inline my-app/src/modules/Home/Home.css:1]: file name can be `my-app/src/modules/Home/style.css`

**Review by mubbsharanwar — COMMENTED** [review]
need to address suggestions.

File: my-app/src/Shared/Subcategory/Subcategory.jsx (hunk 4 above)
Comment (WishaEman) [reply my-app/src/Shared/Subcategory/Subcategory.jsx]: Yes we can do this, should I change it?

Comment (mubbsharanwar) [reply my-app/src/Shared/Subcategory/Subcategory.jsx]: If you can then let's do

**Pull request merged**